packager/file/threaded_io_file.cc

```diff
@@ -60,7 +52,7 @@ bool ThreadedIoFile::Close() {
     result = Flush();
 
   cache_.Close();
-  task_exit_event_.Wait();
+  WaitForSignal(&task_exited_mutex_, &task_exited_);
```

Comment:
No, because WaitableEvent had configurations for whether and how it would be reset.  absl::Notification can never be reset.  Because TaskHandler can be called multiple times (on open and on seek), we need a signal that can be sent/reset more than once.

Reply:
I see.